types/passport-auth-token/tslint.json

```diff
@@ -0,0 +1,18 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
+        "dt-header": false,
```

Comment:
Don't disable any lint rules for new packages, please. Especially since some, like this header rule, are actually critical for publishing.